client/src/locales/translations/en_US.json

```diff
@@ -1038,6 +1039,18 @@
     "showAnyway": "Show anyway",
     "postHidden": "You have chosen to hide this post",
     "unhide": "Unhide",
-    "youReported": "You have reported this post. It will no longer be visible on your feed"
+    "youReported": "You have reported this post. It will no longer be visible on your feed",
+    "buttonRemove": "Remove",
+    "buttonKeep": "Keep",
+    "removePostTitle": "Remove Post Confirmation",
+    "keepPostTitle": "Keep Post Confirmation",
+    "reportDetails": "Additional details provided by users:",
+    "userReported": "Users reported this post for:",
+    "removeReason": "Reason(s) for removing:",
+    "keepPostConfirm": "Are you sure you want to keep this post?",
+    "reportAcceptedTitle": "Report is accepted!",
+    "reportRejectedTitle": "Report is rejected!",
+    "reportAcceptedBody": "This post is succesfully removed!",
+    "reportRejectedBody": "This post is succesfully kept"
```

Comment:
I don't think this copy for a report being rejected makes sense. I don't have enough context for where this copy is being displayed so I'm not sure how this should be rephrased, but maybe it can just be as simple as `This post was not removed`

Reply:
When I wrote this phrase I felt that there should be some better phrase. But I couldn't find too.